INN/umbrella-citylimits
# Make sure series header css only targets special projects landing page

and neighborhoods

## Changes

This pull request makes the following changes:

- Makes all of the custom header styling for landing pages target
```
.neighborhoods-lp .series-header-container,
.cftl-tax-landing-template-series-landing-special-projects-php .series-header-container
``` 
instead of
```
.series-header-container
```
so it only styles headers on the custom special projects landing page template or neighborhoods landing pages

Default (old) template:
![screencapture-citylimits-test-series-election-watch-2018-2020-07-07-16_59_28](https://user-images.githubusercontent.com/18353636/86842477-420ad680-c073-11ea-9e95-c8b782a59a58.png)

Special projects template:
![screencapture-citylimits-test-series-futuremap-2020-07-07-17_00_07](https://user-images.githubusercontent.com/18353636/86842557-5e0e7800-c073-11ea-94de-14d6fb218586.png)

Neighborhood:
![screencapture-citylimits-test-neighborhoods-far-rockaway-2020-07-07-16_58_25](https://user-images.githubusercontent.com/18353636/86842386-1c7dcd00-c073-11ea-8e39-483c0f611a7c.png)

## Why



For #168

## Testing/Questions

Features that this PR affects:

- Custom series header styling


Questions that need to be answered before merging:

- [ ] Is there a better way to target only these two templates?

Steps to test this PR:

1. View a landing page using the default (old) template and make sure its header area is the default Largo landing page header area
2. View a landing page using the new special projects template and make sure its header area is the custom header area we developed
3. View a neighborhood landing page and make sure its header area is also the custom header area we developed
